expanded/experimental-design-750-words.md

```diff
@@ -33,4 +33,6 @@ This is not conceptually difficult.  The MSA transformer was trained on 26 milli
 
 We will then perform codon multiple sequence alignment, and then ancestral sequence reconstruction as described above.  The ancestral sequences will be more uncertain than in the case of antibodies or viral proteins, and we will experiment with different ways to represent this uncertainty.  
 
-We will provide proof of concept by prioritizing alignments based on structural relevance and novelty.  Starting with our viral datasets, we will gradually expand to include additional sequences.  First, we'll focus on FoldSeek clusters containing relevant viral proteins, then move to other clusters that are closely related according to structural similarity metrics such as TM-score and local distance difference test.  We hypothesize that including these related structures will improve performance according to our metrics.  We will track performance improvements as we incorporate additional MSAs.
\ No newline at end of file
+We will provide proof of concept by prioritizing alignments based on structural relevance and novelty.  Starting with our viral datasets, we will gradually expand to include additional sequences.  First, we'll focus on FoldSeek clusters containing relevant viral proteins, then move to other clusters that are closely related according to structural similarity metrics such as TM-score and local distance difference test.  We hypothesize that including these related structures will improve performance according to our metrics.  We will track performance improvements as we incorporate additional MSAs.
+
+_TB: The beauty of ESM is that you can train one giant model on all proteins and then query this model. If I'm following the proposal, you would still need buckets and there would be an "HA" model and a "spike" model. So I'm assuming the outputs would be a bunch of distinctly trained models? Each model on a different FoldSeek cluster? You'd have hundreds or thousands of small(ish) models? Some clarity here would be helpful._
```

Comment:
No, we'd have one big model. I have modified this as follows:

> If it proves useful to add related sequence alignments to our model training for viral proteins, we will take this work to its logical conclusion and develop a single model for all proteins.

...

> We will provide proof of concept by prioritizing alignments to add as training data to this single model based on structural relevance and novelty.

I just don't want to get to the end of a big ESM-scale training exercise and realize that it didn't actually help with our original viral training goals, or that we should have done something different. This strategy will allow us to iterate.